fabtests/component/dmabuf-rdma/fi-rdmabw-xe.c

```diff
@@ -446,6 +454,23 @@ static int init_nic(int nic, char *domain_name, char *server_name, int port,
 		proxy_buf.mrs[nic] = mr;
 	}
 
+	iov.iov_base = sync_buf.xe_buf.buf;
+	iov.iov_len = 4;
+	mr_attr.mr_iov = &iov;
+	mr_attr.iov_count = 1;
+	mr_attr.access = FI_REMOTE_READ | FI_REMOTE_WRITE;
```

Comment:
Why remote read/write for the sync buffer?

Reply:
ah, side effect of copy & paste. will fix.